### PR TITLE
Allow frontend to start displaying questions for ambiguous charges

### PR DESCRIPTION
**Pull request by KentShikama**
This is the backend work for https://github.com/codeforpdx/recordexpungPDX/issues/972.

### CODE REVIEW

File: src/backend/expungeservice/models/helpers/charge_classifier.py [hunk 1]
```diff
@@ -137,7 +155,10 @@ def _subsection_6(section, level, statute):
             "163165",  # ( ineligible if under subection(1)(h) ; Assault in the third degree of a minor 10 years or younger)
         ]
         if section in conditionally_ineligible_statutes:
-            return [Subsection6] + ChargeClassifier._classification_by_level(level, statute)
+            question_string = "TODO: FIXME: Is this charge eligible?"
```
Comment (wittejm) [inline src/backend/expungeservice/models/helpers/charge_classifier.py]: "Does this charge meet the eligibility requirements listed in 137.225(6)? "
options = ["Yes", "No"]

File: src/backend/expungeservice/models/helpers/charge_classifier.py [hunk 2]
```diff
@@ -85,47 +86,64 @@ def _drug_crime(statute, section, name, level):
...
+            return AmbiguousChargeTypeWithQuestion([MarijuanaEligible])
 
     @staticmethod
     def _manufacture_delivery(name, level, statute):
         if any([keyword in name for keyword in ["delivery", "manu/del", "manufactur"]]):
             if "2" in name:
                 if level == "Felony Unclassified":
-                    return [FelonyClassA, FelonyClassB]
+                    question_string = "Was the charge for an A Felony or B Felony?"
+                    options = ["A Felony", "B Felony"]
```
Comment (wittejm) [inline src/backend/expungeservice/models/helpers/charge_classifier.py]: could be a C felony?

Comment (KentShikama) [reply src/backend/expungeservice/models/helpers/charge_classifier.py]: For Schedule 2, I don't think so.

File: src/backend/expungeservice/models/helpers/charge_classifier.py (hunk 1 above)
Comment (KentShikama) [reply src/backend/expungeservice/models/helpers/charge_classifier.py]: We eventually want to split it out but I want to get the "framework" down first.

File: src/backend/expungeservice/crawler/crawler.py [hunk 3]
```diff
@@ -35,7 +35,9 @@ def login(self, username, password, close_session=False) -> bool:
 
         return Crawler.__login_validation(self.response)
 
-    def search(self, first_name, last_name, middle_name="", birth_date="") -> List[AmbiguousCase]:
+    def search(
+        self, first_name, last_name, middle_name="", birth_date=""
+    ) -> Tuple[List[AmbiguousCase], List[Question]]:
```
Comment (wittejm) [inline src/backend/expungeservice/crawler/crawler.py:40]: To support interactive updates as the user submits one or more answers, the search endpoint response needs to also provide the Answers that they have submitted so far. In the first response from the search endpoint, there won't be any answers attached but the response structure is the same. Up to you if you think that's next-PR material.

Comment (KentShikama) [reply src/backend/expungeservice/crawler/crawler.py:40]: The frontend will continue to hit the disambiguate endpoint which returns the questions (with answers) in the record summary object.

**Comment by KentShikama** [comment]
@wittejm I think this is ready but it would be nice to get another pair of eyes on https://github.com/codeforpdx/recordexpungPDX/pull/976/files#diff-1643e6866424073bbb3377c0d2036383 for any leaks.

**Comment by KentShikama** [comment]
There is still a pending TODO for the R&J exception: I'm waiting on @michaelzhang43 for the question. I don't think it blocks the PR merge.

File: src/backend/expungeservice/crawler/crawler.py (hunk 3 above)
Comment (wittejm) [reply src/backend/expungeservice/crawler/crawler.py:40]: oh right!

**Pull request merged**